index.d.ts

```diff
@@ -0,0 +1,36 @@
+interface Card {
+  id: string;
+  text: string;
+  sentiments: {
```

Comment:
this might be answered later in the code but: what's the difference between sentiments and votes?